entries/h/hpe.com.json

```
@@ -0,0 +1,20 @@
+{
+  "HPE": {
+    "domain": "hpe.com",
+    "tfa": [
+      "totp",
+      "custom-hardware",
```

```suggestion
      "u2f",
```
We would just use u2f here since their doucmentation or setup area doesn't specify any proprietary hardware methods.

---

🆗